contracts/src/Token.sol

```diff
@@ -48,147 +32,61 @@ contract Token is IERC20, IERC20Permit {
         name = _name;
         symbol = _symbol;
         decimals = _decimals;
-        GATEWAY = msg.sender;
-        DOMAIN_SEPARATOR = keccak256(
-            abi.encode(
-                TokenLib.DOMAIN_TYPE_SIGNATURE_HASH,
-                keccak256(bytes(_name)),
-                keccak256(bytes("1")),
-                block.chainid,
-                address(this)
-            )
-        );
+        gateway = msg.sender;
     }
 
     modifier onlyGateway() {
-        if (msg.sender != GATEWAY) {
+        if (msg.sender != gateway) {
             revert Unauthorized();
         }
         _;
     }
 
-    /**
-     * @dev Creates `amount` tokens and assigns them to `account`, increasing
-     * the total supply. Can only be called by the owner.
-     *
-     * Emits a {Transfer} event with `from` set to the zero address.
-     *
-     * Requirements:
-     *
-     * - `account` cannot be the zero address.
-     */
```

Comment:
Any reason why the docs were removed?

Reply:
To reduce redundancy. `Token.sol` is just the implementation of the `IERC20.sol` interface, which itself is documented.